ipv-core/src/test/resources/result/ruleOrderMaintained.result.json

```diff
@@ -1 +1,43 @@
-{"resourceType":"Organization","id":"ACME","meta":{"versionId":"1","lastUpdated":"01/2017"},"text":{"status":"generated","div":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"},"extension":[{"url":"http://www.ibm.com/watsonhealth/fhir/extensions/whc-lsf/1.0/studyid","valueString":"XXXXXX"},{"url":"http://www.ibm.com/watsonhealth/fhir/extensions/whc-lsf/1.0/siteid","valueString":"XXXXX"},{"url":"http://www.ibm.com/watsonhealth/fhir/extensions/whc-lsf/1.0/patientid","valueString":"XXXXXXXX"}],"name":"XXXXXXXXXXXXXXXXX","telecom":[{"system":"phone","value":"(+1) 507-867-5309"},{"system":"fax","value":"XXXXXXXXXXXXXXXXX"},{"system":"email","value":"doctor1@hospital1.org"}],"address":[{"line":["X","XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX","XX","XX","XXXXX","XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX","XXXXXXX","XXXXXXXX","XXXXXXXXX"],"city":"Hope","state":"ND","postalCode":"580","country":"USA"}]}
\ No newline at end of file
+{
```

Comment:
There is a test that is failing.  I believe it is because it is expecting this file to be all on one line instead of formatted.  It does a direct compare of the input to the output it generates which is not formatted.